File: mo-common/mo-common-oss/src/main/java/ink/ckx/mo/common/oss/service/OssService.java
package ink.ckx.mo.common.oss.service;

import ink.ckx.mo.common.oss.vo.FileInfoVO;
import org.springframework.web.multipart.MultipartFile;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/14
 */

public interface OssService {

    FileInfoVO uploadFile(MultipartFile file);

    void deleteFile(String key);
}

File: mo-common/mo-common-oss/src/main/kotlin/ink/ckx/mo/common/oss/service/OssService.kt
package ink.ckx.mo.common.oss.service

import ink.ckx.mo.common.oss.vo.FileInfoVO
import org.springframework.web.multipart.MultipartFile

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/14
 */
interface OssService {

    fun uploadFile(file: MultipartFile): FileInfoVO

    fun deleteFile(key: String)
}